documentation/gnmi/design/DASH.json

```diff
@@ -0,0 +1,724 @@
+{
```

Comment:
Please help add more info on what objects can be modified and the expected behaviors. For ACL specifically, what is the expected behavior for existing connections when ACL rules change from “allow” to “deny” that impacts those connections?